runtime/cmake/caches/aix_ppc-64.cmake

```diff
@@ -35,4 +35,6 @@ set(J9VM_GC_IDLE_HEAP_MANAGER OFF CACHE BOOL "")
 set(J9VM_OPT_SWITCH_STACKS_FOR_SIGNAL_HANDLER OFF CACHE BOOL "")
 set(J9VM_THR_ASYNC_NAME_UPDATE OFF CACHE BOOL "")
 
+set(OMR_PORT_NUMA_SUPPORT ON CACHE BOOL "")
```

Comment:
This is also set in the common.cmake which is include after this line, which one wins? Perhaps properties that don't have a common value across all platforms shouldn't be in the common.cmake file, so there is no confusion.

Reply:
Ive tried to remove non-common values from the common.cmake, but evidently some slipped through.  the `set( xx CACHE)` commands only set the value in the cache if it does not already exist. Thus this file takes priority over common.cmake (and command line options can override both files)

Reply:
I think this information should be a comment before the line that includes common.cmake

Reply:
I don't think turning on OMR_PORT_NUMA_SUPPORT for AIX matches current behavior. I see it currently enabled for plinux, xlinux, and Mac, but not other platforms. Other files in this commit turn it on for plinux, xlinux, but not Mac.

Reply:
Fixed

Reply:
@pshipton